vms/spchainvm/key_chain.go

```diff
@@ -28,16 +28,16 @@ type KeyChain struct {
 	Keys   []*crypto.PrivateKeySECP256K1R
 }
 
-// NewKeyChain creates a new keychain for a chain
-func NewKeyChain(networkID uint32, chainID ids.ID) *KeyChain {
-	return &KeyChain{
+// NewKeychain creates a new keychain for a chain
+func NewKeychain(networkID uint32, chainID ids.ID) *Keychain {
+	return &Keychain{
 		chainID: chainID,
 		keyMap:  make(map[[20]byte]int),
 	}
 }
 
 // New returns a newly generated private key
-func (kc *KeyChain) New() *crypto.PrivateKeySECP256K1R {
+func (kc *Keychain) New() *crypto.PrivateKeySECP256K1R {
 	factory := &crypto.FactorySECP256K1R{}
 
 	skGen, _ := factory.NewPrivateKey()
```

Comment:
Should keychain have a log so we can log errors in places like this?

Reply:
It's a bit of a pain to require logging, so I just returned the error instead.